docs/coding-guidelines/coding-style.md

```diff
@@ -36,7 +36,7 @@ The general rule we follow is "use Visual Studio defaults".
 
 An [EditorConfig](https://editorconfig.org "EditorConfig homepage") file (`.editorconfig`) has been provided at the root of the runtime repository, enabling C# auto-formatting conforming to the above guidelines.
 
-We also use the [dotnet-format Tool](https://github.com/dotnet/format) to ensure the code base maintains a consistent style over time, the tool automatically fixes the code base to conform to the guidelines outlined above.
+We also use the [dotnet-format Tool](https://github.com/dotnet/sdk/tree/main/documentation/format) to ensure the code base maintains a consistent style over time, the tool automatically fixes the code base to conform to the guidelines outlined above.
```

Comment:
```suggestion
We also use the [dotnet-format tool](https://learn.microsoft.com/dotnet/core/tools/dotnet-format) to ensure the code base maintains a consistent style over time, the tool automatically fixes the code base to conform to the guidelines outlined above.
```
This is the link to the actively maintained official docs that look better structured to me than the repo docs. It would be best to delete the redundant docs in the SDK repo.

Reply:
I was wondering where should we link this.. This is better, thanks!